Saritasa.Tools.sln

```diff
@@ -32,7 +32,13 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.editorconfig = .editorconfig
 	EndProjectSection
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Saritasa.Tools.EntityFrameworkCore", "src\Saritasa.Tools.EntityFrameworkCore\Saritasa.Tools.EntityFrameworkCore.csproj", "{D80C1322-862D-467A-9735-5474346DCCCB}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Saritasa.Tools.EntityFrameworkCore", "src\Saritasa.Tools.EntityFrameworkCore\Saritasa.Tools.EntityFrameworkCore.csproj", "{D80C1322-862D-467A-9735-5474346DCCCB}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Saritasa.Cake", "Saritasa.Cake", "{46B3292E-FB1B-42C7-8D7D-28904EE6424E}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Saritasa.Cake", "scripts\Modules\Saritasa.Cake\Saritasa.Cake.csproj", "{EA0E235B-0E52-4C8A-9690-2DF720953926}"
```

Comment:
Let's move it to `src` folder? @ko-vasilev ?

Reply:
I think it would be better to have it in `scripts` folder. We can even have a separate solution for it there, or a separate folder in the main saritasa tools solution?